examples/encfs/README.md

```diff
@@ -176,3 +189,23 @@ Alternatively, the whitelisted command in test-encfs-container outputs the follo
 This is a file inside the filesystem.
 This is a file inside the filesystem.
 ```
+
+### Running the ENCFS example with RSA Key with Salt

```

Comment:
This should just be in step 5 instead of at the bottom.

Reply:
I put this here because RSA does not follow the regular oct sequence of operation. So it will be best for users to look at this section at the end.

Reply:
Putting RSA here, means you're putting key generation steps out of order and a user who wants to do it this way will have to scroll to the bottom to find this info.

Reply:
Also the steps are almost identical with the exception of adding the salt and specifying that it's and RSA key